src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -972,6 +979,49 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
                 JoystickImpl::updateConnections();
             break;
         }
+
+        // File drop event
+        case WM_DROPFILES:
+        {
+            const HDROP hDrop = (HDROP)wParam;
+
+            // Generate a FilesDropped event
+            Event event;
+            event.type = Event::FilesDropped;
+
+            // Get the count
+            event.droppedFiles.count = DragQueryFileW(hDrop, 0xFFFFFFFF, NULL, 0);
+            if (event.droppedFiles.count == 0)
+                break;
+
+            // Expand the buffer list by one vector
+            FilesType& files = *m_droppedFiles.insert(m_droppedFiles.end(), FilesType());
```

Comment:
`m_droppedFiles` is a `std::list` of `std::vector`s of `sf::String`s. First append to the list, then resize the newly appended vector.